domain/src/main/java/ru/startandroid/domain/placesmodels/citiesdetailsmodels/Result.kt

```diff
@@ -0,0 +1,21 @@
+package ru.startandroid.domain.models.placesmodel.citiesdetailsmodels
+
+
+data class Result (
```

Comment:
Do we need all these fields in this class? Let's minimise it's count. And remove unused or excess classes.